### DISCUSSION TITLE
Cant get HERO_ROWS to show

### DISCUSSION THREAD

**Discussion by deniax2** [lostb1t/replex#105] [Q&A]
Hi,

I have replex running, and connect to it locally to try it out using URL http://10.1.1.105:3001/web/index.html#!
But for some reason, I am not seeing any difference to the home screen itself.
Being that I do see the homescreen and there are no errors in the log, it seems to me replex is running and rerouting the traffic.

I tried ingognito mode, but still no difference.
I also kept the enviroment as simple as possible for testing:
```
      REPLEX_HOST: http://10.1.1.105:32400/
      REPLEX_HERO_ROWS: movies.recent
      REPLEX_INCLUDE_WATCHED: false
```

But I still dont see any "trending" on the home page. I did have trending before, generated by PMM, but I deleted that and since then also restarted replex.

Anyone have an idea?

**Comment by lostb1t** [comment]
if you want the recent movies row to show as hero use the following value: 
```
REPLEX_HERO_ROWS: home.movies.recent,movies.recent,movie.recentlyadded
```

for collections follow the readme here: https://github.com/lostb1t/replex#mixed-rows
and here
https://github.com/lostb1t/replex#row-style

note theres a cache of 5 minutes so restart replex after making changes

**Reply by PatrickDollard** [reply]
The recent readme doesn't really show how to make collections hero. If the collection is named Popular This Week what would the naming scheme be?

**Reply by lostb1t** [reply]
> The recent readme doesn't really show how to make collections hero. If the collection is named Popular This Week what would the naming scheme be?

https://github.com/lostb1t/replex?tab=readme-ov-file#hub-style

For custom collections you can change the hub style to hero elements by setting the label "REPLEXHERO" on an collection.

**Reply by PatrickDollard** [reply]
Thank you! It's late and I just spent 2 hours skipping over that one line...

**Reply by PatrickDollard** [reply]
One thing that may be nice to add is the ability to choose if they're hero on the main and library hub separately. Something like REPLEXHERO_MAIN, REPLEXHERO_MOVIE, REPLEXHERO_TV.
